.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
[OPTIONAL] It's a good idea that if your IDE generates any configuration files these are added to `.gitignore` in order to keep the repository clean and professional. 

> Remember that adding a file to the .gitignore cannot also be used to remove files that you added using that technique. If you committed a useless file into your repo you need to remove it manually.


[You can read more about to keep a professional repository here.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)